features/volgindicaties.feature

```diff
@@ -0,0 +1,159 @@
+      En gebruiker "A" heeft een volgindicatie voor de persoon met burgerservicenummer "000009945" en einddatum "2021-03-01"
+      En geen enkele andere gebruiker heeft een volgindicatie voor de persoon met burgerservicenummer "000009945"
+      Als gebruiker "A" op 5 maart 2021 een volgindicatie zet op de persoon met burgerservicenummer "000009945" en geen einddatum
+      Dan stuurt de update applicatie een Ap01 bericht met burgerservicenummer "000009945" naar het GBA-V
+
+   Scenario: Een gebruiker heropent een beëindigde volgindicatie die eerder door een andere gebruiker gevolgd werd
+      Gegeven de persoon met burgerservicenummer "999990949" is NIET ingeschreven in deze gemeente
+      En gebruiker "A" heeft een volgindicatie voor de persoon met burgerservicenummer "999990949" en einddatum "2021-03-01"
+      En geen enkele andere gebruiker heeft een volgindicatie voor de persoon met burgerservicenummer "999990949"
+      En de update applicatie heeft op 1 maart 2021 een Av01 bericht met burgerservicenummer "999990949" naar het GBA-V gestuurd
+      Als gebruiker "B" op 5 maart 2021 een volgindicatie zet op de persoon met burgerservicenummer "999990949" en geen einddatum
+      Dan stuurt de update applicatie een Ap01 bericht met burgerservicenummer "999990949" naar het GBA-V
+
+   Scenario: De einddatum van een volgindicatie is vandaag voor een persoon die ook door een andere gebruiker gevolgd wordt
+      Gegeven de persoon met burgerservicenummer "999990962" is NIET ingeschreven in deze gemeente
+      En gebruiker "A" heeft een volgindicatie voor de persoon met burgerservicenummer "999990962" en einddatum "2021-03-01"
+      En gebruiker "B" heeft een volgindicatie voor de persoon met burgerservicenummer "999990962" en einddatum "2021-03-05"
+      En geen enkele andere gebruiker heeft een actieve volgindicatie voor de persoon met burgerservicenummer "999990962"
+      Als de datum vandaag is 1 maart 2021
+      Dan stuurt de update applicatie GEEN Av01 bericht met burgerservicenummer "999990962" naar het GBA-V
+
+   Scenario: De einddatum van een volgindicatie is vandaag en er zijn geen andere actieve volgindicaties op deze persoon
+      Gegeven de persoon met burgerservicenummer "999990962" is NIET ingeschreven in deze gemeente
+      En gebruiker "A" heeft een volgindicatie voor de persoon met burgerservicenummer "999990962" en einddatum "2021-03-01"
+      En gebruiker "B" heeft een volgindicatie voor de persoon met burgerservicenummer "999990962" en einddatum "2021-03-05"
+      En geen enkele andere gebruiker heeft een actieve volgindicatie voor de persoon met burgerservicenummer "999990962"
+      Als de datum vandaag is 5 maart 2021
+      Dan stuurt de update applicatie een Av01 bericht met burgerservicenummer "999990962" naar het GBA-V
+
+   Scenario: De einddatum van een volgindicatie is een datum in het verleden (Av01 wordt maar één keer gestuurd)
```

Comment:
Hoe wordt gevalideerd dat deze scenario correct is geïmplementeerd?

Reply:
Wat is de reden dat dit als scenarion is toegevoegd? Gebeuren er rare dingen in het GBA-V als er meerdere keren een Av01 bericht voor eenzelfde persoon naar het GBA-V wordt gestuurd?

Reply:
dan stuurt GBA-V een foutmelding. Het lijkt me niet de bedoeling dat er berichten gestuurd worden die fouten opleveren.